resources/k8s-artifacts/choreo-connect/choreo-connect-with-apim-ingress.yaml

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: choreo-connect-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
+    kubernetes.io/ingress.class: nginx
+    nginx.ingress.kubernetes.io/ssl-redirect: "false"
+spec:
+  tls:
+  - hosts:
+      - gw.wso2.com
+      - adapter.wso2.com
+      - apim.wso2.com
+    secretName: tls-secret
+  rules:
+    - host: apim.wso2.com
+      http:
+          paths:
+            - path: /
+              pathType: Prefix
+              backend:
+                service:
+                  name: wso2apim
+                  port:
+                    number: 9443
+    - host: gw.wso2.com
+      http:
+          paths:
+            - path: /
+              pathType: Prefix
+              backend:
+                service:
+                  name: router
+                  port:
+                    number: 9095
+    - host: adapter.wso2.com
```

Comment:
Better to remove this.. We can't handle projects coming from two sources.

Reply:
Fixed with https://github.com/wso2/product-microgateway/pull/1910/commits/d2e594743b57d3bc8d7a22127f75e049bc764d2b